lib/ansible/modules/cloud/amazon/s3.py

```diff
@@ -261,139 +273,161 @@
 import traceback
 from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ssl import SSLError
+from ansible.module_utils.basic import AnsibleModule, to_text
+from ansible.module_utils.ec2 import ec2_argument_spec, get_aws_connection_info, boto3_conn, HAS_BOTO3
 
 try:
-    import boto
```

Comment:
Since this module no longer depends on boto, you should update the [unit tests](https://github.com/ansible/ansible/blob/devel/test/units/modules/cloud/amazon/test_s3.py#L4) to reflect that.

You'll also want to remove `boto` from the [unit test requirements](https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt#L1).